src/actions/commands/actions.ts

```diff
@@ -2629,6 +2631,8 @@ class CommandGoToDefinition extends BaseCommand {
       vimState.cursorStopPosition = Position.FromVSCodePosition(vimState.editor.selection.start);
     }
 
+    vimState.globalState.jumpTracker.recordJump(jumpFrom, Jump.fromStateNow(vimState));
```

Comment:
We should already be doing this here: https://github.com/VSCodeVim/Vim/blob/master/src/mode/modeHandler.ts#L400. 

Is there a bug somewhere else then?